### PR TITLE
feat(Bbox): allow negative and tuple or array amt to grow bounding box

### PR DESCRIPTION
**Pull request by jingpengw**

### CODE REVIEW

File: cloudvolume/lib.py [hunk 1]
```diff
@@ -633,8 +634,11 @@ def volume(self):
...
     return (self.minpt + self.maxpt) / 2.0
 
-  def grow(self, amt):
-    assert amt >= 0
+  def grow(self, amt: Union[int, tuple, list]):
+    if isinstance(amt, tuple) or isinstance(amt, list):
+      assert len(amt) == 3
+      amt = np.asarray(amt)
+
     self.minpt -= amt
```
Comment (william-silversmith) [inline cloudvolume/lib.py:642]: It looks like `shrink` has a check for valid Bbox that `grow` doesn't. Should this be incorporated?

Comment (william-silversmith) [reply cloudvolume/lib.py:642]: Should shrink take negative factors too?

Comment (jingpengw) [reply cloudvolume/lib.py:642]: Oh, I did not notice that there is a shrink function!
I just added another function called adjust. please take a look.

Comment (william-silversmith) [reply cloudvolume/lib.py:642]: This seems good, but shouldn't shrink be the negative of grow?

Comment (jingpengw) [reply cloudvolume/lib.py:642]: good point! did not realize that it should be negative!

Comment (jingpengw) [reply cloudvolume/lib.py:642]: I have corrected this in the following commit, please take a look.

**Comment by william-silversmith** [comment]
Okay LGTM!

**Comment by william-silversmith** [comment]
Thank you Jingpeng!

**Pull request merged**